packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -195,23 +195,41 @@ protected override MethodProvider[] BuildMethods()
             return [.. methods];
         }
 
-        private MethodProvider BuildExplicitFromClientResult()
+        internal MethodProvider BuildExplicitFromClientResult()
```

Comment:
Instead of making the method internal lets just find it from the list since its only used in tests